addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -654,6 +654,8 @@ def create(self, vals_list):
         allocations = super(HolidaysAllocation, self.with_context(mail_create_nosubscribe=True)).create(vals_list)
         allocations._add_lastcalls()
         for allocation in allocations:
+            if self._context.get('form_view_ref') == 'hr_holidays.hr_leave_allocation_view_form_dashboard':
```

Comment:
Seems like a big bidouille, didn't you find another way to do this without context key ?